packages/react-storage/src/components/StorageBrowser/composables/DataTable/dataCells/DateDataCell.tsx

```diff
@@ -6,18 +6,19 @@ import { STORAGE_BROWSER_BLOCK_TO_BE_UPDATED } from '../../../constants';
 export interface DateDataCellProps {
   content: {
     date?: Date;
+    text?: string;
```

Comment:
Nit: Could this be `displayText` to distinguish the purpose of the prop a bit more? (Similar to the value and displayValue props for number cells)

Reply:
Sure I think that's ok